dev-tools/cloud/Makefile

```
@@ -36,7 +36,6 @@ cloud-clean: ## Clean cloud deployment
 	@cd ${TERRAFORM_PATH}; terraform destroy -auto-approve
 
 .PHONY: cloud-get-fleet-url
-cloud-get-fleet-url:  ## Clean cloud deployment
-	$(eval FLEET_SERVER_URL := $(shell terraform output --state=${TERRAFORM_PATH}/terraform.tfstate fleet_url))
-	@echo '${FLEET_SERVER_URL}'
-
+cloud-get-fleet-url:  ## Get Fleet URL from this deployment
+	@# required to add "tr" command since terraform outputs the values with double quotes
+	@terraform output --state=${TERRAFORM_PATH}/terraform.tfstate fleet_url | tr -d \"
```

Can `--raw` help here?
```suggestion
	@terraform output --raw --state=${TERRAFORM_PATH}/terraform.tfstate fleet_url
```

---

Sure, that is the format I was looking for, thanks!